### DISCUSSION TITLE
I Just Bought the Complete Time Circuits Display Kit - All Aluminum!

### DISCUSSION THREAD

**Discussion by WMasterJedi** [CircuitSetup/Time-Circuits-Display#20] [General]
Yesterday I Bought the Complete Time Circuits Display Kit - All Aluminum!
I did choose the more expensive All Aluminum version because I thought even though it makes it quite a lot more expensive if I'm going to buy a great quality Time Circuit Display, I should go all out and get the best one!

So, I can't wait for it to arrive (it will take longer for me as I'm in the UK) and start assembling the Time Circuit Display!

**Comment by CircuitSetup** [comment]
awesome! I'll get your kit in the mail hopefully this week!

**Reply by WMasterJedi** [reply]
Thanks!
